flake.nix

```diff
@@ -30,6 +30,7 @@
             version = "0.0.0";
             vendorHash = builtins.readFile ./tsnsrv.sri;
             src = with p; lib.sourceFilesBySuffices (lib.sources.cleanSource ./.) [".go" ".mod" ".sum"];
+            ldflags = [ "-s" "-w" ];
```

Comment:
This is failing the alejandra check: https://github.com/boinkor-net/tsnsrv/actions/runs/7681585200/job/20934931225?pr=96#step:6:17 - could you run`nix fmt` and commit that?

Reply:
alejandra has in my opinion no sense of aesthetics but sure if you like it...

Reply:
```suggestion
            ldflags = ["-s" "-w"];
```